fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -548,7 +561,21 @@ public TMasterOpResult proxyExecute(TMasterOpRequest request) {
                 UUID uuid = UUID.randomUUID();
                 queryId = new TUniqueId(uuid.getMostSignificantBits(), uuid.getLeastSignificantBits());
             }
-            executor.execute(queryId);
+            Context extractedContext = Telemetry.getOpenTelemetry().getPropagators().getTextMapPropagator()
```

Comment:
The extractedContext comes from the follower node and you should use `Span.fromContext(extractedContext).getSpanContext().isValid()` to check whether it is valid. If valid then initialize tracer.

Reply:
Done.